### ISSUE TITLE
main.js missing from repo

### ISSUE THREAD

**Issue by KY4U** [Azure-Samples/storage-blob-php-webapplication#1]
The public/js/main.js file that is referenced in the index.php can not be located in the repository.

**Comment by ondrejfiala** [comment]
You are right, they omitted it. Actually, "main.js" is noted in the ".gitignore" file.

**Comment by KY4U** [comment]
@paulbatum , @berberich , @bketelsen , @johndeu 

Can any of you push the public/js/main.js file to this repository.  This is the second time I came back around to this code only to find the files is still missing.   I would like to test the script in all of its glory!
